source/Ship.cpp

```diff
@@ -201,9 +201,6 @@ void Ship::Load(const DataNode &node)
 				armament.AddGunPort(hardpoint, outfit);
 			else
 				armament.AddTurret(hardpoint, outfit);
-			// Print a warning for the first hardpoint after 32, i.e. only 1 warning per ship.
-			if(armament.Get().size() == 33)
-				child.PrintTrace("Warning: ship has more than 32 weapon hardpoints. Some weapons may not fire:");
```

Comment:
Why remove this warning entirely if a limit still exists?

Reply:
Ah, yeah, good point.

Reply:
Fixed with 918d1ad